docs/integrations/sources/hubspot.md

```diff
@@ -34,5 +50,5 @@ The connector is restricted by normal Hubspot [rate limitations](https://legacyd
 
 * For api key auth, in Hubspot, for the account to go settings -&gt; integrations \(under the account banner\) -&gt; api key. If you already have an api key you can use that. Otherwise generated a new one.
```

Comment:
As far as I can tell, this connector/airbyte would need to implement oauth for a client to be able to obtain oauth credentials. Since we don't do this yet, can you remove the oauth section from the docs and create an issue to support it once we have oauth? blocked on this issue: #768

Reply:
ok https://github.com/airbytehq/airbyte/issues/2449